server/gameconnection.py

```diff
@@ -294,14 +295,24 @@ async def handle_game_result(self, army, result):
         except (KeyError, ValueError):  # pragma: no cover
             self._logger.warning("Invalid result for %s reported: %s", army, result)
 
-    async def handle_operation_complete(self, army, secondary, delta):
-        if not int(army) == 1:
+    async def handle_operation_complete(
+        self,
+        primary_objectives_complete: typing.Any,
+        secondary_objectives_complete: typing.Any,
```

Comment:
Is there really a point in using `Any` here? Omitting the type signature would have the same effect I think. Also the rest of the codebase imports the needed types from `typing` and uses them without the `typing.` prefix, so lets at least keep that consistent.

Reply:
Given that the function could end up getting a variety of inputs here I personally think it's a useful hint to future developers that they can't assume uniform input types?

Reply:
Maybe. If you're gonna do it for one `handle_*` function you should probably do it for all of them though for consistency.